itest/rust/src/builtin_tests/script/script_instance_tests.rs

```diff
@@ -31,6 +32,38 @@ impl IScriptExtension for TestScript {
     unsafe fn instance_create(&self, for_object: Gd<Object>) -> *mut c_void {
         create_script_instance(TestScriptInstance::new(self.to_gd().upcast()), for_object)
     }
+
+    fn editor_can_reload_from_file(&mut self) -> bool { unreachable!() }
+    fn get_base_script(&self) -> Option<Gd<Script>> { unreachable!() }
+    fn get_global_name(&self) -> StringName { unreachable!() }
+    fn inherits_script(&self, _script: Gd<Script>) -> bool { unreachable!() }
+    fn get_instance_base_type(&self) -> StringName { unreachable!() }
+    unsafe fn placeholder_instance_create(&self, _for_object: Gd<Object>) -> *mut c_void { unreachable!() }
+    fn instance_has(&self, _object: Gd<Object>) -> bool { unreachable!() }
+    fn has_source_code(&self) -> bool { unreachable!() }
+    fn get_source_code(&self) -> GString { unreachable!() }
+    fn set_source_code(&mut self, _code: GString) { unreachable!() }
+    fn reload(&mut self, _keep_state: bool) -> Error { unreachable!() }
+    fn get_documentation(&self) -> Array<Dictionary> { unreachable!() }
+    fn has_method(&self, _method: StringName) -> bool { unreachable!() }
+    #[cfg(since_api = "4.2")]
+    fn has_static_method(&self, _method: StringName) -> bool { unreachable!() }
```

Comment:
This is likely going to be a maintenance nightmare as new APIs are added in the future, but yeah 😬